### PR TITLE
Database

### PR DESCRIPTION
**Pull request by ylysenkova**

### CODE REVIEW

File: db/schema.sql [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+create table if not exists jland_site.user
+(
+	id serial not null
+		constraint user_id_pkey
+			primary key,
+	first_name varchar(255) [],
+	last_name varchar(255) [],
+	photo bytea
+)
+;
```
Comment (groksana) [inline db/schema.sql]: I don't see presentation_plan

File: db/schema.sql [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+       TABLESPACE = pg_default
+       LC_COLLATE = 'en_US.UTF-8'
+       LC_CTYPE = 'en_US.UTF-8'
+       CONNECTION LIMIT = -1;
+
+-- create schema
+CREATE SCHEMA jland_site
+  AUTHORIZATION admin;
+
+GRANT ALL ON SCHEMA jland_site TO admin;
```
[the remark this answers is not in the document]
Comment (ylysenkova) [reply db/schema.sql]: removed

File: db/schema.sql [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+-- create tables
+create table if not exists jland_site.conference
+(
+	id serial not null
+		constraint conference_id_pkey
+			primary key,
```
[the remark this answers is not in the document]
Comment (ylysenkova) [reply db/schema.sql]: Not fully understand what is not OK in this case

File: db/schema.sql [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+			description varchar
+	date date,
+	place varchar(255) []
```
[the remark this answers is not in the document]
Comment (ylysenkova) [reply db/schema.sql]: renamed

File: db/schema.sql [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+)
+;
+
+create table if not exists jland_site.event
+(
+  id serial not null
```
[the remark this answers is not in the document]
Comment (ylysenkova) [reply db/schema.sql]: removed all "not null" for SERIAL

File: db/schema.sql [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+		constraint event_id_pkey
+			primary key,
+			conference_id integer not null
+		constraint event_conference_id_fkey
+			references jland_site.conference,
+  name varchar(255) [],
```
[the remark this answers is not in the document]
Comment (ylysenkova) [reply db/schema.sql]: Agree. It's  not needed

File: db/schema.sql [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+  type varchar(255) [],
+  start_time time,
+  end_time time,
+  order integer
+)
+;
+
+create table if not exists jland_site.presentation
+(
+	id serial not null
```
[the remark this answers is not in the document]
Comment (ylysenkova) [reply db/schema.sql]: Fixed

File: db/schema.sql [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+	name varchar(255),
+	user_id integer,
+	constraint presentation_user_id_fkey
+			references jland_site.user,
+	point varchar(255) [],
+	order integer
+)
+;
+
+create index if not exists fki_presentation_fk
```
[the remark this answers is not in the document]
Comment (ylysenkova) [reply db/schema.sql]: renamed

File: db/schema.sql [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+	on jland_site.presentation (event_id)
+;
+
+create table if not exists jland_site.user
+(
+	id serial not null
```
[the remark this answers is not in the document]
Comment (ylysenkova) [reply db/schema.sql]: removed

**Pull request merged**